Tools/CMake/AMReXThirdPartyLibraries.cmake

```
@@ -6,9 +6,29 @@ if (AMReX_HDF5)
        set(HDF5_PREFER_PARALLEL TRUE)
     endif ()
     find_package(HDF5 1.10.4 REQUIRED)
+
     if (AMReX_MPI AND (NOT HDF5_IS_PARALLEL))
-       message(FATAL_ERROR "\nHDF5 library does not support parallel I/O")
+       if (CMAKE_VERSION VERSION_LESS 3.27)
+	  # The detection in earlier versions of cmake may not be reliable.
+          # So we will try to do it ourselves.
```

Formatting and reference:
```suggestion
	      # The detection in earlier versions of cmake may not be reliable.
          # So we will try to do it ourselves. Work-around for:
          # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8234
```